src/utils/calculateTooltipOffset.test.js

```diff
@@ -2,14 +2,14 @@ import calculateTooltipOffset from './calculateTooltipOffset';
 
 describe('Tooltip offset calculator', () => {
   it('should calculate offset for a 2 hour meeting', () => {
-    expect(calculateTooltipOffset(2)).toBe(-54);
+    expect(calculateTooltipOffset(2)).toBe(-54.5);
```

Comment:
Because of the change to the calculate width function, these results will change. We are calculating the width when we calculate the tooltip offset, and then divide that number by two. The results vary depending on the length of the meeting.